lib/marc/datafield.rb

```diff
@@ -96,7 +102,7 @@ def initialize(tag, i1=' ', i2=' ', *subfields)
 
     def to_s
       str = "#{tag} "
-      str += "#{indicator1}#{indicator2} " 
+      str += "#{indicator1}#{indicator2} "
       @subfields.each { |subfield| str += subfield.to_s }
       return str
```

Comment:
As above, replace `return str` with `str`.  Consider `@subfields.inject` and this method might be a one liner.